src/app/pages/Help.svelte

```diff
@@ -0,0 +1,88 @@
+<style>
+  .help-page {
+    width: 75%;
+    margin: 1.5em auto auto;
+  }
+
+  .header {
+    font-size: var(--xxl);
+  }
+
+  .nav-back {
+    margin: 1rem 0 0 1rem;
+  }
+
+  .paragraph-container {
+    margin: 1rem 0 1rem 0;
+  }
+
+  .link {
+    margin-left: 1rem;
+  }
+</style>
+
+<main>
+  <nav class="nav-back"><a href="/" class="button button--blue">Back</a></nav>
+  <article class="help-page">
+    <header>
+      <h1 class="header">What Do I Upload?</h1>
+      <p>
+        Predictive Text Studio uses the spreadsheet you provide to customize the
+        words in your keyboard's dictionary. The "Word" column contains
+        frequently used words in your language that you would like to add to
+        your dictionary. The "Count" column gives the amount of times or
+        frequency that a word appears in your text.
+      </p>
+    </header>
+    <section>
+      <h2>Excel</h2>
+      <div class="paragraph-container">
+        Create an Excel file with "Word" in the A column and "Count" in the B
+        column. You may upload the Excel file directly to the Upload Area.
+      </div>
```

Comment:
Use a `<p>` here, and then apply the margins to the `p`.


```suggestion
      <p>
        Create an Excel file with "Word" in the A column and "Count" in the B
        column. You may upload the Excel file directly to the Upload Area.
      </p>
```